src/drivers/intel/cavs/timer.c

```diff
@@ -43,7 +43,7 @@ int64_t platform_timer_set(struct timer *timer, uint64_t ticks)
 	if (ticks > shim_read64(SHIM_DSPWC))
 		shim_write64(SHIM_DSPWCT0C, ticks);
 	else
-		shim_write64(SHIM_DSPWCT0C, shim_read64(SHIM_DSPWC) + 1);
+		shim_write64(SHIM_DSPWCT0C, shim_read64(SHIM_DSPWC) + 100);
```

Comment:
Can we make this a macro at the top of the file (obvious for anyone debugging). I suppose long term we could make this variable depending if LP ckock is running etc.

Reply:
Done.